stack.yaml

```diff
@@ -61,8 +61,5 @@ extra-deps:
   - tasty-hedgehog-1.0.0.1
   - Unique-0.4.7.6
 
-  # Windows only
-  - Win32-2.5.4.1
-
```

Comment:
Looking at the Stack snapshot we're using: https://www.stackage.org/lts-13.26, it seems that Win32 is pinned to 2.6.1.0, while you're introducing a `Win32 >= 2.6.2.0` constraint in the cabal file. So I don't think this is buildable on Windows using Stack.

Unfortunately, our CI doesn't catch this.

Can you try building it with Stack on Windows?

That's the only thing blocking this PR for me.

Reply:
I get many errors `the stack configuration has no specified version` on windows, if I don't add the 3 dependencies above.

Reply:
Those three dependencies are already pinned in lts-13.26, so it is strange that you have to add them. I'm thinking too that this is a local problem. I wouldn't feel comfortable adding them.

Reply:
@mrBliss do you think this pr is ready now? I can't find what I do wrong in my local machine.

Reply:
Yeah, let's merge it. At least we know that it doesn't break things on Unix :slightly_smiling_face:.
Building with Cabal works for you on Windows, right?

Reply:
Yes cabal builds fine.